src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -465,9 +466,16 @@ static int open_device(const char* path, bool try_load_module = false)
     if (r < 0) {
       cerr << "rbd-nbd: failed to load nbd kernel module: " << cpp_strerror(-r) << std::endl;
       return r;
+    } else {
+      load_module = true;
     }
     nbd = open(path, O_RDWR);
   }
+
+  if (nbds_max && try_load_module && !load_module) {
+    cerr << "rbd-nbd: nbds_max param doesn't work, because nbd module already loaded." << std::endl;
```

Comment:
@liupan1111 @trociny How about something simple like "rbd-nbd: ignoring kernel module parameter options: nbd module already loaded" if either ```nbds_max``` or ```max_part``` are specified and the module is already open.

Also, can you open a bug tracker ticket for this PR?